### ISSUE TITLE
Epicea: Recover lost changes should be more robust

### ISSUE THREAD

**Issue by demarey** [guillep/pharo-core#4876]
In pharo 7, we used to have a "Recovert lost changes" dialog when opening a Pharo image after an image crash. It is not there anymore in Pharo 8.

To try:
- create a class + methods
- make the image crash (on OS X full window -> sleep -> wake up)
- reopen the image

@tinchodias any idea?

**Comment by tinchodias** [comment]
I didn't check in latest 8.0, but have used it before. I suspect there is something tricky in the Epicea's Recover Lost Changes that confuses here:

Epicea creates a ombu log file with the first change/event to log. The recovering check doesn't realize there are changes to recover if you just start Pharo, do changes and quit without save. You need to do some changes, SAVE the image, then do more changes and finally quit without save. 

Could you check, please? Maybe it's time to revise this behavior of the recovering check...

**Comment by demarey** [comment]
@tinchodias Indeed, you're right. With a previous save, the recovery dialog appears.
Maybe you could just check if the last event in the latest ombu log file is alreay applied in the image to determine if there are changes to recover?

**Comment by tinchodias** [comment]
Maybe... The entry point, to try alternatives, is `EpLostChangesDetector class>>#startUp:`.

But not sure we want something so smart every time an image starts... This is a scenario that made me give up in have a smarter detection of lost changes:

"Hi, I'm Pharo.image starting and the pointer to current Epicea log is nil. What should I do?"

I thought a bit in checking files, but disk operations are slow, then discarded it.

**Comment by tinchodias** [comment]
I guess I should check at this issue
